.circleci/config.yml

```
@@ -232,7 +232,7 @@ jobs:
       - image: circleci/ruby:2.4
     working_directory: /tmp/deploy
     environment:
-      - DISTROS: "trusty xenial bionic el6 el7"
+      - DISTROS: "xenial bionic el6 el7"
```

This uncovered oversight in previous EL8 work.

Deployment should include `el8` as well:
```suggestion
      - DISTROS: "xenial bionic el6 el7 el8"
```

---

I noticed that, but that's outside the scope of this PR.

---

cc @punkrokk

---

Is there another PR then or do I need to make one?

---

There is no PR for this kind of fix yet

---

Captured in #4899.